src/member/routes.ts

```diff
@@ -30,11 +31,15 @@ export const buildGetMemberStorageRoute = () =>
 
 // Storage files
 export const buildGetMemberStorageFilesRoute = (
-  currentPage: number,
-  pageSize: number,
-) =>
-  `${MEMBERS_ROUTE}/current/storage/files?page=${currentPage}&pageSize=${pageSize}`;
+  pagination: PaginationParams,
+) => {
+  const searchParams = new URLSearchParams();
 
+  searchParams.set('currentPage', (pagination.page ?? 1).toString());
+  searchParams.set('pageSize', (pagination.pageSize ?? 10).toString());
```

Comment:
Why rename the parameters ? I think it is important to be consistent with other endpoints using the same mechanism. If we need to update the specifications file to reflect that, then this is not an issue.